monai/fl/client/monai_algo.py

```diff
@@ -494,6 +494,11 @@ def initialize(self, extra=None):
             BundleKeys.DATASET_DIR, default=ConfigItem(None, BundleKeys.DATASET_DIR)
         )
 
+        if self.multi_gpu:
```

Comment:
is it possible to achieve this in https://github.com/Project-MONAI/model-zoo/blob/dev/models/spleen_ct_segmentation/configs/logging.conf cc @Nic-Ma

Reply:
Ok, lets merge this for now but ideally the detailed logging logic should be agnostic to the trainer..

Reply:
There's ways of changing the logger's [Formatter class](https://docs.python.org/3/library/logging.html#logging.Formatter) to write the process and thread ID with each message. This doesn't state explicitly which rank you're on but you can work it out from the PID values or at least know which process is which.

Reply:
I feel it's hard for common users to find out the matching process ID of ranks..
CC @holgerroth @wyli .

Thanks.

Reply:
It is so you're not really finding that out, just differentiating which process wrote which line in the log, though typically the children PIDs  will be only a few values more than the parent's PID and in order.